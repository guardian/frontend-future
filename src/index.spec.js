import index from './index';

document.body.innerHTML = `
    <style class="_styletron_hydrate_"></style>
    <div id="app"></div>
`;

it('renders correctly on the server', () => {
    expect(index()).toMatchSnapshot();
});

it('renders correctly in the browser', () => {
    global.BROWSER = true;
    global.SERVER = false;
    expect(index()).toMatchSnapshot();
});
