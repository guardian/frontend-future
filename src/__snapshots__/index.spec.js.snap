// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly in the browser 1`] = `"<h1 class=\\"a b c\\">Hello from Preact!<sub class=\\"a d\\">and Styletron!</sub></h1>"`;

exports[`renders correctly on the server 1`] = `
Object {
  "css": "<style class=\\"_styletron_hydrate_\\">.c:hover{background-color:#00456e}.a{color:#005689}.b{font-size:16px}.d{font-size:12px}</style>",
  "html": "<h1 class=\\"a b c\\">Hello from Preact!<sub class=\\"a d\\">and Styletron!</sub></h1>",
}
`;
